kmod/src/ioctl.h

```diff
@@ -442,6 +442,9 @@ struct scoutfs_ioctl_alloc_detail_entry {
  * and destination inodes.  If an error is returned then partial
  * progress may have been made and inode fields may have been updated.
  *
+ * If MB_STAGE flag is set, destination range is allowed to be in an
+ * offline extent.
```

Comment:
We should describe that the destination range *has* to be an offline extent, what we'll return if it's not, and we should add a description of requiring data_version as well when staging.